components/brave_wallet_ui/page/screens/send/send/send.tsx

```diff
@@ -410,6 +411,8 @@ export const Send = (props: Props) => {
         wrapContentInBox={true}
         cardWidth={sendContainerWidth}
         noMinCardHeight={true}
+        hideNav={props.isAndroid}
+        hideHeader={props.isAndroid}
```

Comment:
Lets add `isAndroid` to the `props` deconstruction on line `83 - 90` then we can just use `isAndroid` instead of `props.isAndroid`.

```suggestion
        hideNav={isAndroid}
        hideHeader={isAndroid}
```